v1/plugin/plugin_test.go

```diff
@@ -239,7 +239,7 @@ func TestMakeTLSConfig(t *testing.T) {
 		Convey("plugin lib should use TLS config requiring verified clients and specific cipher suites", func() {
 			config := tlsSetupInstance.makeTLSConfig()
 			So(config.ClientAuth, ShouldEqual, tls.RequireAndVerifyClientCert)
-			So(config.PreferServerCipherSuites, ShouldEqual, true)
+			So(len(config.CipherSuites), ShouldBeGreaterThan, 0)
```

Comment:
`PreferServerCipherSuites` is deprecated and removed. the recommended approach is to use CipherSuites